packages/renderer/src/lib/preferences/PreferencesRegistriesEditing.svelte

```diff
@@ -0,0 +1,459 @@
+<style>
+.dropdown:hover .dropdown-menu {
+  display: block;
+}
+</style>
```

Comment:
I think we should move the style at the end of the file to keep
script, markup, style order

https://svelte.dev/docs#component-format

Reply:
removed after refactoring